src/renderer/components/notifications/show-checked-error.injectable.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { JsonApiErrorParsed } from "../../../common/k8s-api/json-api";
+import loggerInjectable from "../../../common/logger.injectable";
+import type { Disposer } from "../../utils";
+import type { CreateNotificationOptions } from "./notifications.store";
+import showErrorNotificationInjectable from "./show-error-notification.injectable";
+
+export type ShowCheckedErrorNotification = (message: unknown, fallback: string, opts?: CreateNotificationOptions) => Disposer;
+
+const showCheckedErrorNotificationInjectable = getInjectable({
```

Comment:
I don't understand the concept of `checkedErrorNotification`. Seems like something that shouldn't exist.

Reply:
It is a helper around `showErrorNotification`, maybe it should be called `checkedShowErrorNotification` instead. It does some additional logic to make sure that the value that gets through to "reasonable" with a declarative fallback message for better UX.